apps/storybook/src/LineVis.stories.tsx

```diff
@@ -31,114 +29,17 @@ const errorsArray = ndarray(
   dataArray.shape
 );
 
-const Template: Story<LineVisProps> = (args) => {
-  const {
-    dataArray,
-    domain: storyDomain,
-    scaleType,
-    errorsArray,
-    auxiliaries,
-  } = args;
-
-  // If story doesn't provide `domain`, compute it automatically
-  const auxArrays = (auxiliaries || []).map(({ array }) => array);
-  const domain =
-    storyDomain ||
-    getCombinedDomain([
-      getDomain(dataArray, scaleType, errorsArray),
-      ...getDomains(auxArrays, scaleType),
-    ]);
-
-  return <LineVis {...args} domain={domain} />;
-};
-
-export const Default = Template.bind({});
-Default.args = {
-  dataArray,
-  domain,
-};
-
-export const Domain = Template.bind({});
-Domain.args = {
-  dataArray,
-  domain: [-100, 100],
-};
-
-export const Abscissas = Template.bind({});
-Abscissas.args = {
-  dataArray,
-  domain,
-  abscissaParams: {
-    value: abscissas,
-  },
-};
-
-export const DescendingAbscissas = Template.bind({});
-DescendingAbscissas.args = {
-  dataArray,
-  domain,
-  abscissaParams: {
-    value: [...abscissas].reverse(),
-  },
-};
-
-export const ErrorBars = Template.bind({});
-ErrorBars.args = {
-  dataArray,
-  errorsArray,
-  showErrors: true,
-};
-
-export const AuxiliaryArrays = Template.bind({});
-AuxiliaryArrays.args = {
-  dataArray: primaryArray,
-  auxiliaries: [{ array: secondaryArray }, { array: tertiaryArray }],
-};
-
-export const AuxiliaryLabels = Template.bind({});
-AuxiliaryLabels.args = {
-  dataArray: primaryArray,
-  auxiliaries: [
-    { label: 'secondary', array: secondaryArray },
-    { label: 'tertiary', array: tertiaryArray },
-  ],
-};
-
-export const AuxiliaryErrors = Template.bind({});
-AuxiliaryErrors.args = {
-  dataArray: primaryArray,
-  errorsArray,
-  auxiliaries: [
-    { label: 'secondary', array: secondaryArray, errors: errorsArray },
-    { label: 'tertiary', array: tertiaryArray },
-  ],
-  showErrors: true,
-};
-
-export const IgnoreValue = Template.bind({});
-IgnoreValue.args = {
-  ...Default.args,
-  ignoreValue: (val) => val % 5 === 0,
-};
-
-export const TypedArrays = Template.bind({});
-TypedArrays.args = {
-  dataArray: toTypedNdArray(primaryArray, Float32Array),
-  errorsArray: toTypedNdArray(errorsArray, Float32Array),
-  auxiliaries: [
-    { label: 'secondary', array: toTypedNdArray(secondaryArray, Float32Array) },
-  ],
-  showErrors: true,
-};
-
-export { Template as LineVisTemplate };
-export default {
+const meta = {
   title: 'Visualizations/LineVis',
   component: LineVis,
-  excludeStories: ['LineVisTemplate'],
-  parameters: { layout: 'fullscreen', controls: { sort: 'requiredFirst' } },
   decorators: [FillHeight],
+  parameters: {
+    layout: 'fullscreen',
+    controls: { sort: 'requiredFirst' },
+  },
   args: {
+    dataArray,
+    domain: undefined, // compute dynamically in each story based on scale type, errors and auxiliaries
```

Comment:
This is an important change. I'm letting the domain be computed dynamically in the stories by default to avoid errors when the user changes the scale type with the controls. Users can still set a `domain` of their own choosing with the controls if they want.